src/python_testing/TC_TIMESYNC_2_2.py

```diff
@@ -78,10 +78,9 @@ async def test_TC_TIMESYNC_2_2(self):
         compare_time(received=utc_dut, utc=th_utc, tolerance=tolerance)
 
         self.print_step(5, "Read time source")
-        if self.check_pics("TIMESYNC.S.A0002"):
-            source = await self.read_ts_attribute_expect_success(endpoint=endpoint, attribute=attributes.TimeSource)
-            if utc_dut_initial is NullValue:
-                asserts.assert_equal(source, Clusters.Objects.TimeSynchronization.Enums.TimeSourceEnum.kAdmin)
+        source = await self.read_ts_attribute_expect_success(endpoint=endpoint, attribute=attributes.TimeSource)
```

Comment:
you'll need to do the same thing as above, where you first check if this attribute is supported, otherwise this is changing the test logic.

Reply:
I have updated this section of the code in TC_TIMESYNC_2_2 to match with TC_TIMESYNC_2_1 test module now.